### PR TITLE
Use react-dom to reference DOM

### PR DESCRIPTION
**Pull request by pukapukan**
To suppress warning introduced by react v0.14

### CODE REVIEW

**Comment by romanenko** [comment]
@okonet Yes, you are right. But this requires react `>= 0.14`, and it means that there will be a need to release a new version with breaking changes.

**Comment by okonet** [comment]
Well, I don't think you can do `require('react-dom')` with React 0.13, so this should be a major release either way. Or am I missing something?

**Comment by pukapukan** [comment]
@okonet is right.  I was naively thinking react-dom would be compatible with previous versions of react.  I'll close this one as @okonet said - it'll be a major release to upgrade to react add dependency to v0.14.  Cheers

**Comment by paramaggarwal** [comment]
Keeping this open to eventually support React 0.14.

File: index.js [hunk 1]
```diff
@@ -116,7 +117,7 @@ var Dropzone = React.createClass({
   },
 
   open: function() {
-    var fileInput = React.findDOMNode(this.refs.fileInput);
+    var fileInput = ReactDOM.findDOMNode(this.refs.fileInput);
```
Comment (newoga) [inline index.js]: Do we actually need ReactDOM here?

We should test to see if `this.refs.fileInput` is enough... That way this project doesn't have a dependency on `ReactDOM`

**Comment by newoga** [comment]
FYI, just let a comment on a line. `this.refs.fileInput` should actually return the DOM node in 0.14.0. This is described here: https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs

File: index.js (hunk 1 above)
Comment (okonet) [inline index.js]: Yes, I'm aware of that. Thanks!

Comment (newoga) [inline index.js]: Okay cool :)

**Comment by okonet** [comment]
Can we have it rewritten without using ReactDOM and `findDOMNode`?

**Comment by okonet** [comment]
I'll merge #84 if it's okay for everyone.

**Comment by pukapukan** [comment]
Lovely go ahead please

**Comment by okonet** [comment]
Ok cool. Closing this one in favor of #84

**Pull request closed without merging**